<!doctype html>
<meta charset="utf-8">
<meta name="viewport" content="width=device-width, initial-scale=1">
<meta property="og:title" content="Expository Representation Learning of Biomedical Data">
<meta property="og:url" content="http://sbbi-panda.unl.edu/bibm2019/">
<meta property="og:description" content="The International Workshop on Expository Representation Learning of Biomedical Data @ BIBM 2019">
<meta property="og:image" content="http://sbbi-panda.unl.edu/bibm2019/static/og.jpg">
<script src="https://use.typekit.net/bmw2mzd.js"></script>
<script>try{Typekit.load({ async: true });}catch(e){}</script>
<link rel="stylesheet" href="./static/normalize.min.css?h=fe4f3f56">
<link rel="stylesheet" href="./static/style.css?h=3a66124f">
<title>Expository Representation Learning of Biomedical Data</title>
<body>
  <header>
    <h2 class="center">
      <div class="red"><b>The International Workshop on Expository Representation Learning of Biomedical Data</b></div>
      <div class="gray">November 18-21, 2019, San Diego</div>
    </h2>
  </header>
  <nav data-gumshoe-header>
    <ul data-gumshoe class="flex">
      <li><a href="http://ieeebibm.org/BIBM2019/" target="_blank">BIBM 2019</a></li>
      <li><a data-scroll href="/#about">Home</a></li>
      <li><a data-scroll href="/#organizer">Organizers</a></li>
      <li><a data-scroll href="/#program-committee">Program Committee</a></li>
      <li><a data-scroll href="/#important-date">Important Dates</a></li>
      <li><a data-scroll href="/#accepted-papers">Accepted Papers</a></li>
      <li><a data-scroll href="/#schedule">Workshop Schedule</a></li>
    </ul>
  </nav>
  
<div id="about">
  <p><a href="https://wi-lab.com/cyberchair/2019/bibm19/scripts/submit.php?subarea=S22&amp;undisplay_detail=1&amp;wh=/cyberchair/2019/bibm19/scripts/ws_submit.php"><center><h2><font color="blue"><b>Online Submission</b></font></h2></center></a></p>
<p>In today's world data is produced at a mind-boggling volume and
pace. According to marketing reports 90 percent of the data in the
world was produced within the past two years. With the recent advances
in high-throughput technologies, OMICS data in the biomedical field
would be one of the biggest contributor to this everlasting
growth. These large volumes of data are widely accessible; however,
they are mostly unstructured while having the potential to deepen our
understanding of complex problems from disease outbreak to disease
diagnosis and treatment. Statistical and machine learning methods can
help expose the hidden value of unstructured data, improving
prediction accuracy, and build predictive models that go beyond human
performance. Recently, there is a surge of new techniques in the
context of discovering latent structure in high dimensional data such
as causal relations in multifaceted regulation networks, genotype and
phenotype association, genomics signatures, and risk factors
etc. However, it can still be a long way to obtain satisfactory
results in scalable learning with increasing data size and
complexity. Furthermore, a wide variety of technologies produce
heterogeneous data shedding light on different aspects of complex
biological systems. Emergence of a more complete picture of biological
systems depends on successful methods for integration of data from
these different perspectives.</p>
<p>With this workshop, we aim to encourage researchers to develop new
methodologies, analytical models, and high-throughput computing
workflow that best utilize various types of biomedical data in ways
that meaningful structures present but hidden in data can be revealed.</p>
<h3>Topics of Interest</h3>
<ul>
<li>Data-centric models for diagnosis and classification of complex human disease such as cancers and obesity</li>
<li>Learning based prediction models for drug response assessment</li>
<li>Integrative models for identification of dynamic and multi-level biological interaction networks (proteins, non-coding genes, metabolites). </li>
<li>Integrative models for discovery of disease associated cell communication</li>
<li>Complex phylogenetic models to elucidate cancer genome evolution</li>
<li>High-throughput computing workflow development for data mining, visualization, and interpretation</li>
<li>High-throughput computing workflow development for biomedical image processing classification</li>
<li>Integrative models for visualization and analysis of data in neuroscience</li>
<li>Information fusion based modeling for reverse engineering biological networks</li>
<li>Inferring genomic signatures from sequencing and expression data</li>
<li>Modeling and simulation of complex biological processes in high-throughput computing environments</li>
</ul>

</div>
<div id="organizer" class="flex">
  <div class="col-100">
    <h2>Organizers</h2>
    <ul>
<li>Juan Cui, PhD, Department of Computer Science and Engineering, University of Nebraska-Lincoln, USA</li>
<li>Zeynep Hakguder, Department of Computer Science and Engineering, University of Nebraska-Lincoln, USA</li>
<li>Haluk Dogan, Department of Computer Science and Engineering, University of Nebraska-Lincoln, USA</li>
</ul>

  </div>
</div>
<div id="program-committee" class="flex">
  <div class="venue-text col-100">
    <h2>Program Committee</h2>
    <h4><strong>TBD</strong></h4>

  </div>
</div>
<div id="important-date" class="flex">
  <div class="venue-text col-100">
    <h2>Important Dates</h2>
    <ul>
<li><strong>September 20, 2019:</strong> Due date for full workshop papers submission</li>
<li><strong>October 15, 2019:</strong> Notification of paper acceptance to authors</li>
<li><strong>November 1, 2019:</strong> Camera-ready submission of accepted papers</li>
<li><strong>November 18-21, 2019:</strong> Workshops</li>
</ul>

  </div>
</div>
<div id="accepted-papers" class="flex">
  <div class="venue-text col-100">
    <h2>Accepted Papers</h2>
    <h4><strong>TBD</strong></h4>

  </div>
</div>

<div id="schedule">
  <h2>Workshop Schedule</h2>
  <h4><strong>TBD</strong></h4>

</div>


  

  <script src="./static/header.js?h=6ac8d165"></script>
  
<script src="./static/smooth-scroll.min.js?h=f748edd9"></script>
<script src="./static/gumshoe.min.js?h=570e9b3c"></script>
<script src=""></script>
<script src="./static/root.js?h=eeb412b2"></script>

</body>
